### DISCUSSION TITLE
Navigation menu vs Tabs

### DISCUSSION THREAD

**Discussion by nicoladaniello** [radix-ui/primitives#1989] [Help]
I'm probably overthinking this, but I'm not sure whether I should use Tabs or Navigation Menu for the layout I'm trying to achieve.

The component I need looks like tabs, but it doesn't need to control TabContent areas as clicking on a tab will redirect to a different page. The other page will keep the same layout with the tabs but change the content below it.

This is similar to how the Vercel dashboard works. Here is an example:

![Platforms-on-Vercel](https://user-images.githubusercontent.com/15842507/221444563-d69bb1d1-c272-4246-b3ed-b726a7c4e8e0.png)

Would you suggest using the Tabs primitive with links as triggers, or a navigation menu?

**Comment by benoitgrelard** [comment]
> The component I need looks like tabs, but it doesn't need to control TabContent areas as clicking on a tab will redirect to a different page. The other page will keep the same layout with the tabs but change the content below it.

If I understand this correctly, I think I'd lean towards NavigationMenu here in this case.
